lib/ansible/utils/plugin_docs.py

```diff
@@ -65,7 +65,7 @@ def add_fragments(doc, filename, fragment_loader):
             fragment_var = 'DOCUMENTATION'
 
         if fragment_class is None:
-            raise AnsibleAssertionError('fragment_class is None')
+            raise AnsibleAssertionError('Could not find documentation fragment %s' % fragment_slug)
```

Comment:
@flowerysong Thanks for the contribution. Could you please add a test case for this change ?

Reply:
No.